frontend-public/src/styles/components/MineInfo.scss

```diff
@@ -0,0 +1,38 @@
+.mine-title {
+  font-size: $title-h1;
+  padding-bottom: $default-padding-xl;
+}
+
+.mine-info-padding {
+  padding-top: $default-padding-xxl;
+  padding-bottom: 70px;
+}
+
+.mine-info-header-row {
+  background-color: $lightest-grey;
+  box-shadow: $default-box-shadow;
+  margin: 24px 0px;
+  h4 {
+    padding-top: $default-padding-lg;
+    padding-bottom: $default-padding-md;
+  }
+}
+
+.user-upload-document-btn {
+  color: $font-colour !important;
```

Comment:
Concerned about the number of `!importants` used here, 
should set defaults for ant designs buttons instead and use those instead of these custom classes.

Reply:
There are a lot of different button styles on this site so a lot of them I need styling that isn't overriding the ant styles. This one though had to be important as it was ignoring it otherwise.

Reply:
Is the goal to override this class to use this font colour everywhere, in all instances? If not, we should probably try to get more specific with the selector instead of using `!important`. If so, we should have this in the `theme.scss` or `themeOverrides.scss`. I'm not sure which file is more appropriate

Reply:
I have refactored as much as I can into the theme overrides and am using primary, secondary, tertiary, and ghost buttons.